pabi_hr_advance_status_report/wizard/pabi_hr_advance_status_report_wizard.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+import time
+
+class PabiHrAdvanceStatusReportWizard(models.TransientModel):
+
+    _name = 'pabi.hr.advance.status.report.wizard'
+
+    run_date = fields.Date(string="Run Date", default=time.strftime("%Y-%m-%d"))
```

Comment:
@phongyanon this should be, (for current date, context_today is better (it follow the correct time zone)
```
run_date = fields.Date(
  string="Run Date",
  default=fields.Date.context_today(self),
)
```